packages/framework/ai-collab/api-report/ai-collab.alpha.api.md

```diff
@@ -4,6 +4,47 @@
 
 ```ts
 
+// @alpha
+export function aiCollab<TSchema extends ImplicitFieldSchema>(options: AiCollabOptions<TSchema>): Promise<AiCollabSuccessResponse | AiCollabErrorResponse>;
+
+// @alpha
+export interface AiCollabErrorResponse {
+    errorMessage: "tokenLimitExceeded" | "tooManyErrors" | "tooManyModelCalls" | "aborted";
+    status: "failure" | "partial-failure";
+    tokenUsage: TokenUsage;
+}
+
+// @alpha
+export interface AiCollabOptions<TSchema extends ImplicitFieldSchema> {
+    dumpDebugLog?: boolean;
+    finalReviewStep?: boolean;
+    limiters?: {
+        abortController?: AbortController;
+        maxSequentialErrors?: number;
+        maxModelCalls?: number;
+        tokenLimits?: TokenUsage;
+    };
+    // (undocumented)
```

Comment:
Please be sure to document any undocumented API members here before merging. Anything we package-export should be documented.

Reply:
All members have been documented as of commit e835b1c037abfbab72df0c82e66781f54a4bdfe9